test/ui/test/button-link-edit-browse.jsx

```diff
@@ -0,0 +1,59 @@
+import ButtonLinkEditBrowse from '../../../src/components/buttons/button-link-edit-browse.jsx';
+
+(function() {
+	'use strict';
+
+	var assert = chai.assert;
+	var Simulate = ReactTestUtils.Simulate;
+	var TestUtils = ReactTestUtils;
+	var documentBrowseLinkCallback;
+	var documentBrowseLinkUrl = 'http://alloyeditor.com/some/location';
+
+	describe('ButtonLinkEditBrowse', function() {
+		before(function(done) {
+			documentBrowseLinkCallback = sinon.spy();
+
+			Utils.createAlloyEditor.call(this, done, {
+				documentBrowseLinkCallback: documentBrowseLinkCallback,
+				documentBrowseLinkUrl: documentBrowseLinkUrl
```

Comment:
Can write this as:

```js
Utils.createAlloyEditor.call(this, done, {
  documentBrowseLinkCallback,
  documentBrowseLinkUrl,
});
```

Reply:
Yes, this is related to my previous comment but sounds good to me